tests/setup_local_db/test_data_atlas.sql

```diff
@@ -27,6 +27,14 @@ values
     (4,'Test cohort4','Extra Larger cohort')
 ;
 
+insert into atlas.cohort_definition_details
+(id,expression,hash_code)
+values
+    (1,'{"expression" : "1" }',987654321),
+    (2,'{"expression" : "2" }',1234567890),
+    (3,'{"expression" : "3" }', NULL)
```

Comment:
please add/extend a unit test that asserts on the presence of the right value in the new `Expression` attribute of the CohortDefinition struct.